Header header
uint8 type
float64 omega
float64 velocity
float64 theta